components/ProgressBar/ProgressBar.module.scss

```diff
@@ -0,0 +1,56 @@
+.progressStepper {
+    position: relative;
+    margin-top: 1rem;
+    width: 100%;
+
+    .progressBar {
+        position: relative;
+        height: 6px;
+        width: 97%;
+        margin: auto;
+        background-color: #ddd;
+        border-radius: 3px;
+        margin-bottom: 20px;
+
+        .progressBarFill {
+            height: 100%;
+            background-color: #a19366;
+            border-radius: 3px;
+            transition: width 0.4s ease;
```

Comment:
can we try ease-out and a longer animation for this? The current one seems a bit rushed